heartbeat/monitors/monitor.go

```diff
@@ -35,7 +35,7 @@ import (
 )
 
 // ErrMonitorDisabled is returned when the monitor plugin is marked as disabled.
-var ErrMonitorDisabled = fmt.Errorf("monitor not loaded, plugin is disabled")
+var ErrMonitorDisabled = fmt.Errorf("monitor not loaded, disabled with enabled: false")
```

Comment:
IMO this is a bit more confusing because of using two somewhat contradictory words. Although it makes sense, I'd suggest something like:

```

monitor not loaded because its 'enabled' key was set to false
```

IMO it's better to have a lengthier sentence but make it a bit more explicit.

Sorry for the bikeshedding, it LGTM, but just thought I'd mention it in case you thought it was a good idea.

Reply:
I like this better because we're avoiding the inclusion of a `:` character as well.

Reply:
++ much improved